01_foregrounds.ipynb

```diff
@@ -107,11 +107,11 @@
     "            dtype=None,\n",
     "            verbose=False,\n",
     "        )\n",
-    "        emission_map[emission_map == hp.UNSEEN] = np.nan\n",
+    "        emission_map[emission_map == 0] = np.nan\n",
     "        if emission == \"CMB_tensor_to_scalar_ratio\":\n",
-    "            # input maps are simulated with r=1\n",
+    "            # input maps are simulated with r=3e-3\n",
     "            # tensor-to-scalar ratio is defined in the power spectrum, we need to weight the maps with the square root\n",
-    "            weight = np.sqrt(weight)\n",
+    "            weight = np.sqrt(weight/3e-3)\n",
```

Comment:
@keskitalo can you doublecheck here? so I have input sims with `r=3e-3`, so if the user chooses `3e-3` in the tool, I don't make any change to the map.
If they choose e.g. 3e-5, I divide the map by 10.

Reply:
Sounds correct.